physical_infra_provider_configuration/automation/topics/creating_instance_lenovo_Xclarity_class.adoc

```diff
@@ -0,0 +1,41 @@
+==== Creating a new instance to the LenovoXclarity class
+
+1.  Select your domain created and drill down Your Domain name→ &nbsp;System → Event → EmsEvent →LenovoXclarity and select **(LenovoXclarity)**.
+
+2.  Click **(Configuration)**, then click **(Add a new Instance)**.
+
+3.  Enter with the name
+
+    a.  The name must be equal to event name that will be managed, because the Cloud Forms match the instance with the event by the name.
+    
+    b.  Find the name of the event that you wish in this link **<TO DO>**
+    
+    c.  Copy the name of the event and past in the field’s name in Cloud Forms **(ex. FQXHMTS0003G)**.
+
+4.  Add the path to the handlers copied in the field’s **rel3**.
```

Comment:
What is rel3?

Reply:
Hi Stephen, sorry about the "&nbsp", I think that it is a problem in my text editor. Firstly thank you for your review and the rel3 is the name of the field (Relationship three). If you have any suggestions about that, please let me know.

Reply:
@CharlleDaniel So _rel3_ is the name of a field in the ManageIQ UI?

Reply:
yes